### PR TITLE
Content Pages: Build root About page's highlights and mobile sections

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
app-content-pages

## Linked Issue and/or Talk Post
Toward: https://github.com/zooniverse/front-end-monorepo/issues/838
Figma: https://www.figma.com/file/zN9InDg8CTk2vnWp406moK/About-Pages?node-id=445%3A249&mode=dev

## Describe your changes

Build Highlights component and Mobile component for their sections in the root About page.

## How to Review

Run app-content-pages locally and visit https://local.zooniverse.org:3000/about/about#highlights and https://local.zooniverse.org:3000/about/about#mobile.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [x] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

### CODE REVIEW

**Comment by goplayoutside3** [comment]
@seanmiller26 Question about the tooltip designed for the books - 

Our Grommet styling library has an easy implementation Tip component that looks like this:

Screenshot 2024-02-22 at 3 51 34 PM

It's difficult to show an arrow on the left of the tooltip, so I left it as just the text like in the above screenshot for now.

My question is about smaller viewports and touch screens. In the video below you can see I resize my browser to a smaller width, and the tooltip no longer shows up because it would be off screen. Is there an alternative solution to using a tooltip for this information? Tooltips also aren't very accessible by screen-readers because they require hovering over an icon to get the info. The Classifier has an existing tooltip and it lives in the Banner componen, but does not have great responsiveness or accessibility: [storybook here](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/banners-alreadyseenbanner--default).

My suggestion would be to just include "We do not make any profit through sales of the hard copies of the books" as text alongside "Available for purchase". Maybe the purchase link could be a bit bigger with additional text below it in smaller font size?

https://github.com/zooniverse/front-end-monorepo/assets/23665803/dcfde394-fb8f-4260-a089-a91321b8b3c5

**Comment by seanmiller26** [comment]
I'm not sure I understand the limitation here. Can we use the Grommet option of appearing below the icon like this: https://storybook.grommet.io/?path=/story/controls-tip-simple--simple

**Comment by goplayoutside3** [comment]
@seanmiller26 I realized that the tooltip icon wasn't interacting at certain screen widths because the 'Contact Us' section below it has a hidden margin that allows navigation to its heading (like on all About pages with sidebars).

That's my bad 🤦🏼‍♀️ I fixed the interaction bug and also placed the tooltip's content below it so we don't get any hanging carets incorrectly positioned.

It looks like this and is ready for code review!
desktop

Also works by tapping a mobile screen:
mobile

File: packages/app-content-pages/src/screens/About/components/Highlights.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        {t('AboutPage.highlights.paragraphs.two')}
+      </Paragraph>
+      <Box
+        justify='center'
+        gap='xsmall'
+        margin={{ top: '30px' }}
+        direction='row'
+      >
+        <Image
+          alt=''
```
Comment (mcbouslog) [inline packages/app-content-pages/src/screens/About/components/Highlights.js:48]: Currently in PFE it looks like we have alt text for the Highlights book images along the lines of "Zooniverse Highlights Book Cover 2022." Is that something we could add for these images as well?

File: packages/app-content-pages/src/screens/About/components/Mobile.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        alignSelf='center'
+        weight='normal'
+        margin='0'
+        textAlign='center'
+      >
+        {t('AboutPage.mobile.subheading')}
+      </Heading>
+      <Box align='center'>
+        <Image
+          alt=''
```
Comment (mcbouslog) [inline packages/app-content-pages/src/screens/About/components/Mobile.js:20]: Similar to Highlights comment, could we add an alt description here?

**Review by mcbouslog — APPROVED** [review]
Looks great! Really cool to see the about pages coming together 🎉

**Comment by seanmiller26** [comment]
Very minor styling to touch up here -

Screen Shot 2024-03-04 at 7 34 20 PM

Can we make sure the tops and bottoms of the books are aligned?

File: packages/app-content-pages/src/screens/About/components/Highlights.js (hunk 1 above)
Comment (goplayoutside3) [reply packages/app-content-pages/src/screens/About/components/Highlights.js]: Yes! I wasn't sure if we wanted a screen reader to read decorative images like these, but I added alt translation strings.

**Comment by goplayoutside3** [comment]
> Can we make sure the tops and bottoms of the books are aligned?

Yes! Done

**Pull request merged**